react-api-client/src/camera/useUpdateCameraImageSettings.ts

```diff
@@ -0,0 +1,66 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { createCameraImageSettings } from '@opentrons/api-client'
+import { OT_SYSTEM_CAMERA } from '@opentrons/shared-data'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type {
+  UseMutateFunction,
+  UseMutationOptions,
+  UseMutationResult,
+} from 'react-query'
+import type {
+  CameraImageSettings,
+  CameraImageSettingsResponse,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export type UseUpdateCameraImageSettingsMutationResult = UseMutationResult<
+  CameraImageSettingsResponse,
+  AxiosError<ErrorResponse>,
+  CameraImageSettings
+> & {
+  updateCameraImageSettings: UseMutateFunction<
+    CameraImageSettingsResponse,
+    AxiosError<ErrorResponse>,
+    CameraImageSettings
+  >
+}
+
+const cameraId = OT_SYSTEM_CAMERA
+
+export function useUpdateCameraImageSettings(
```

Comment:
```suggestion
export function useCreateCameraImageSettings(
```
Because the underlying request is a `POST` instead of a `PUT`, it's more accurate to say "create" here. Doing so also enables the hook name to mirror the underlying network request naming scheme.